api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -67,6 +73,19 @@ public void setUserDAO(UserDAO dao) {
 		this.dao = dao;
 	}
 	
+	
+	/**
+	 * @return the validTime for which the password reset activation key will be valid
+	 */
+	private int getValidTime() {
+		final int validTime = Context.getAdministrationService().getGlobalProperty("security.validTime") == null
```

Comment:
- Please use the constant here
- You need to take care of when the value is an empty string or a white space, use StringUtils.isBlank() because it takes care of all these